docs/ExpectAPI.md

```diff
@@ -646,6 +647,45 @@ test('the house has my desired features', () => {
 });
 ```
 
+### `toHaveProperty(propPath, value)`
```

Comment:
i actually think `propPath` is much easier to understand if you're new to it. both `ref` and `key` words are very ambiguous and cam mean a million things, which will make it hard for beginners to understand what it actually stands for. But with the prop path it's a little more straightforward.

what do you thingk @thymikee ?